scripts/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "USAGE $0 ENVIRONMENT_NAME"
+    exit 1
+fi
+
+environment_name="$1"
+hostname="${environment_name}.libero.pub"
+ssh "$hostname" /usr/local/bin/deploy browser="$(scripts/latest-revision.sh git@github.com:libero/browser)"
```

Comment:
the simplest form `/usr/local/bin/deploy` could take is
```
git clone git@github.com:libero/my-sample-configuration
cd my-sample-configuration
./update.sh $@ # updates .env with passed versions, runs `docker-compose up`
```
I can inline this here to be more opinionated. Opinions?

Reply:
(meaning: executing this steps through SSH)